### PR TITLE
Fix extra monitor memory leak

### LINKED ISSUE #796: Extra monitor core runs out of SDRAM on download
The extra monitor core ran out of memory apparently.  It would be hoped that this wouldn't happen.  It would be good to work out why it does.

[error.log](https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/files/6377280/error.log)

### PR DESCRIPTION
**Pull request by rowleya**
Fixes #796
Tested by http://apollo.cs.man.ac.uk:8080/blue/organizations/jenkins/Integration%20Tests/activity?branch=fix_extra_monitor_memory_leak

### CODE REVIEW

**Comment by rowleya** [comment]
I don't know for sure that this fixes the leak, but it does make sure that a free happens where it wasn't before (and the pointer was lost), and the tests pass...

File: c_common/models/system_models/src/extra_monitor_support.c [hunk 1]
```diff
@@ -1818,7 +1823,9 @@ static void data_out_speed_up_command(sdp_msg_pure_data *msg) {
                     data_out_transaction_id, message->transaction_id);
             return;
         }
+#if 0
```
Comment (Christian-B) [inline c_common/models/system_models/src/extra_monitor_support.c:1826]: Why do if #0
Would it not be better to either of
1. Just comment out
2. And if def with a flag that could be set at compile time
3. Just delete they easy enough to add back in as as when required.

**Review by Christian-B — APPROVED** [review]
Aprroved with or without the ugly if #0

File: c_common/models/system_models/src/extra_monitor_support.c (hunk 1 above)
Comment (rowleya) [reply c_common/models/system_models/src/extra_monitor_support.c:1826]: Agreed, but will leave for a later PR to do this properly with log_debug etc.

**Pull request merged**